src/main/kotlin/com/kotlin/boardproject/domain/post/domain/NormalPost.kt

```diff
@@ -1,10 +1,11 @@
-package com.kotlin.boardproject.model
+package com.kotlin.boardproject.domain.post.domain
 
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.enums.NormalType
-import com.kotlin.boardproject.common.enums.PostStatus
-import com.kotlin.boardproject.common.exception.ConditionConflictException
-import com.kotlin.boardproject.dto.post.normalpost.EditNormalPostRequestDto
+import com.kotlin.boardproject.global.enums.ErrorCode
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end